apps/challenges/challenge_config_utils.py

```diff
@@ -277,6 +277,12 @@ def validate_challenge_config_util(
         message = "Please add the challenge title"
         error_messages.append(message)
 
+    # Check for challenge domain
+    challenge_domain = yaml_file_data.get("domain")
+    if not challenge_domain or challenge_domain not in ["CV", "NLP", "RL"]:
+        message = "Please select a valid challenge domain"
```

Comment:
Improve this message to inform that it cannot be empty and it should either of those three values.